pkg/api/v20230701preview/openshiftcluster_validatestatic.go

```diff
@@ -35,12 +36,16 @@ func (sv openShiftClusterStaticValidator) Static(_oc interface{}, _current *api.
 	sv.domain = domain
 	sv.requireD2sV3Workers = requireD2sV3Workers
 	sv.resourceID = resourceID
+	var architectureVersion api.ArchitectureVersion
 
 	oc := _oc.(*OpenShiftCluster)
 
 	var current *OpenShiftCluster
 	if _current != nil {
+		architectureVersion = _current.Properties.ArchitectureVersion
 		current = (&openShiftClusterConverter{}).ToExternal(_current).(*OpenShiftCluster)
+	} else {
+		architectureVersion = version.InstallArchitectureVersion
```

Comment:
No need for this, just the arch version to what's here.  
```suggestion
```

Reply:
Then can make this consistent throughout all the api versions that support this.